simulations/uniswap/plotting.py

```diff
@@ -21,6 +33,6 @@ def plot_results(results: List[List[Tuple[int, int]]]):
 
     fig, ax = plt.subplots(figsize=(6, 3))
     ax.plot(prices[:, 0], label="Uniswap price")
-    ax.plot(prices[:, 1], label="External market price")
+    ax.plot(prices[:, 1], "--", label="External market price")
```

Comment:
What does this do (for my own knowledge)

Reply:
plots a dashed line